openy_data_wrapper/src/DataWrapper.php

```diff
@@ -217,6 +218,44 @@ public function getBranchPins($id = NULL) {
     return $this->getPins('branch', $id);
   }
 
+  /**
+   * Gets value for image field of template.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   An object of the Membership node.
+   *
+   * @return mixed|null
+   *   A value of the field.
```

Comment:
It should be array|null isn't it?
 * @return array|null
 *   A render array for the image or NULL if no image is found.

Reply:
fixed